ADOPTERS.md

```diff
@@ -0,0 +1,11 @@
+# Who's using Open Cluster Management
+
+### Adopters
+
+Below is a list of companies and projects that have adopted Open Cluster Management(OCM). If you have been using OCM and your name is not on this list, send us a PR!
+
+- [Alibaba Cloud](https://www.alibabacloud.com/)
+- [Ant Group](https://www.antgroup.com/)
+- [Red Hat](https://www.redhat.com/)
+- [Tencent Cloud](https://cloud.tencent.com/)
+- [Xiao Hong Shu](https://www.xiaohongshu.com/)
```

Comment:
@promacanthus would you confirm if it is ok?

Reply:
> @promacanthus would you confirm if it is ok?

Yes, it's all good.